CONTRIBUTING.md

```diff
@@ -4,40 +4,39 @@ Contributions are always welcome, no matter how large or small. Before contribut
 please read the [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Setup
-
 * Install Go 1.16
```

Comment:
Maybe adding a link to the official doc or a good tutorial can help.
I was lost already at this step the first time install go in mac was not working, since you have to run your project in a specific folder or set your env